src/main/java/com/theateam/waaprojectspringbackend/controller/CustomersController.java

```diff
@@ -53,15 +60,45 @@ public void updateOffer(@PathVariable Long offerId, @RequestBody UpdateOfferDto
     }
 
     @GetMapping("/saved-properties")
-    public String getSavedProperties() {
-        return "Greetings from Customers GET saved-properties!";
+    public ResponseEntity<List<String>> getSavedProperties() {
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        String userEmail = authentication.getName();
+        Optional<User> user = userService.getUserByEmail(userEmail);
+
+        if(user!=null){
+            List<String> savedPropertyIds = user.get().getSavedPropertyIds();
+            return ResponseEntity.ok(savedPropertyIds);
+        }else {
+            return ResponseEntity.status(HttpStatus.INTERNAL_SERVER_ERROR).body(null);
```

Comment:
Can you just return like "return savedPropertIds", its enough if we get 200 on success, no need to handle anything else.